addons/survey/wizard/survey_invite_views.xml

```diff
@@ -5,27 +5,30 @@
             <field name="name">survey.invite.view.form</field>
             <field name="model">survey.invite</field>
             <field name="arch" type="xml">
-                <form string="Compose Email" class="o_mail_composer_form">
+                <form string="Compose Email" class="o_mail_composer_form" style="padding-bottom: 0px;">
                     <group col="1">
                         <group col="2">
                             <field name="survey_access_mode" invisible="1"/>
                             <field name="survey_users_login_required" invisible="1"/>
                             <field name="survey_users_can_signup" invisible="1"/>
-                            <field name="survey_id" readonly="context.get('default_survey_id')"/>
+                            <field name="survey_id" readonly="context.get('default_survey_id')" invisible="1"/>
                             <field name="existing_mode" widget="radio" invisible="1" />
                             <field name="render_model" invisible="1"/>
-                            <field name="survey_start_url" label="Public share URL" readonly="1" widget="CopyClipboardChar"
-                                 attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"
-                                 class="mb16"/>
+                            <label class="o_survey_label_survey_start_url" for="survey_start_url" string="Survey Link"/>
+                            <field name="survey_start_url" nolabel="1" readonly="1" widget="CopyClipboardChar"
+                                 attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"/>
+                            <field string="Send by Email" name="send_email" widget="boolean_toggle"
+                                   attrs="{'invisible':[('survey_access_mode', '!=', 'public')]}"/>
                             <field name="partner_ids"
                                 widget="many2many_tags_email"
                                 placeholder="Add existing contacts..."
-                                context="{'force_email':True, 'show_email':True, 'no_create_edit': True}"/>
+                                context="{'force_email':True, 'show_email':True, 'no_create_edit': True}"
+                                attrs="{'invisible': [('send_email', '=', False)]}"/>
                             <field name="emails"
                                 attrs="{
-                                    'invisible': [('survey_users_login_required', '=', True)],
+                                    'invisible': ['|', ('survey_users_login_required', '=', True), ('send_email','=',False)],
                                 }"
-                                placeholder="Add a list of email recipients (separated by commas, semicolons or line breaks). They will not be converted into contacts."/>
+                                placeholder="e.g.  'Rick Sanchez' &lt;rick_sanchez@example.com&gt;, hictor_vugo@example.com"/>
```

Comment:
Not sure why there is a double-space between the "e.g" and the rest, even if not a big deal.